wire-ios-data-model/Source/Core Crypto/CoreCryptoKeyProvider.swift

```diff
@@ -129,3 +166,35 @@ struct LegacyCoreCryptoKeychainItem: KeychainItemProtocol {
         ]
     }
 }
+
+struct CoreCryptoKeychainItemV2: KeychainItemProtocol {
+
+    var id: String {
+        "com.wire.mls.key"
+    }
+
+    var keychainServiceName: String {
+        "wire.com"
+    }
+
+    var getQuery: [CFString: Any] {
+        [
+            kSecClass: kSecClassGenericPassword,
+            kSecAttrService: keychainServiceName,
+            kSecAttrAccount: id,
+            kSecReturnData: true
+        ]
+    }
+
+    func setQuery(value: some Any) -> [CFString: Any] {
+        [
+            kSecClass: kSecClassGenericPassword,
```

Comment:
The new key is `keychainServiceName` because we can use attributes as `kSecAttrComment` or `kSecAttrLabel` to provide more info about the key.